### ISSUE TITLE
Mouse wheel issue with specific mice

### ISSUE THREAD

**Issue by immortalx74** [ocornut/imgui#2581]
Version: 1.68
Branch: master
Back-ends: imgui-SFML
Operating System: Windows 10

ISSUE: Please note, this doesn't seem to be a Dear ImGui issue, rather a problem with specific mice. I just thought it would be good to let you know.

On a minimal application (just displaying the demo window), using a Microsoft mouse (Wireless mouse 5000), the mouse wheel produces no output unless I do a very fast 360 degrees rotation.
This mouse works 100% fine with all other applications. I can confirm that other mice work as expected in ImGui.

Sometime ago I encountered the same issue (again with this mouse AND another Microsoft mouse) on a FlexGrid control that someone has written. I informed the author and he was able to fix it, referring to an old MSDN post. All I know (from his response) is that "some mice require an accumulation of the incoming (smaller) Rotation-delta-values."

I believe the MSDN post was this: [https://social.msdn.microsoft.com/Forums/en-US/1deb5f7e-95ee-40ac-84db-58d636f601c7/wmmousewheel-possible-issues-in-vista-solution?forum=gametechnologiesgeneral](url)

**Comment by ocornut** [comment]
The way the code is written in most back-ends and in imgui should make that work unless the delta are extremely small (because there's rounding involved in some operations).

As a quick test I just divided the wheel delta by /10 in imgui_impl_win32.cpp
`io.MouseWheel += 0.10 * (float)GET_WHEEL_DELTA_WPARAM(wParam) / (float)WHEEL_DELTA;`
And it still works (although of course much slower).

Could you test if your setup works with the three main back-ends included in imgui codebase? 
aka the win32, glfw, and sdl back-end?
There's a possibility this could be a bug of the SFML back-end.

Also with your current back-end, try to print out the value of `io.MouseWheel` after calling `NewFrame()` and see what sorts of value do you get in it.

**Comment by immortalx74** [comment]
You were right Omar, it's something with SFML.
1) I tested the other back-ends and they work absolutely fine!
2) I followed the ImGui-SFML author's own tutorial which uses ImGui::SFML::Update, that itself calls NewFrame(). I printed out the value of io.MouseWheel after that and I get all zeros, EXCEPT when I do a very fast wheel rotation, with which I get 1 or -1 and no in-between values.

Again thank you very much!
(Should I close this now?)
